src/npctrade.cpp

```diff
@@ -674,6 +676,10 @@ bool npc_trading::trade( npc &np, int cost, const std::string &deal )
     //               np.volume_carried() - np.volume_capacity() );
     np.drop_invalid_inventory();
 
+    if( get_option<bool>( "AIM_TRADE" ) ) {
```

Comment:
Can we pass the `deal` string to create_advanced_inv and include it in the header?

Reply:
Fix in latest commit. Thanks for pointing this out